server/src/models/domainEvent.model.ts

```diff
@@ -45,7 +45,7 @@ export class DomainEvent extends Entity {
     required: true,
     postgresql: {
       columnName: 'created_at',
-      dataType: 'timestampz',
+      dataType: 'timestamptz',

```

Comment:
I want to learn from this: what's the difference between these two data type?

Reply:
@dadiorchen it was giving me an unknown type error when migrating postgres data base.